### ISSUE TITLE
Scrollbar arrow buttons are missing in Package Sources VS Options for Win 11

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#12849]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

Version 17.8.0 Preview 2.0 [34020.76.main]

### Worked before?

Yes, not sure

### Impact

It bothers me. A fix would be nice

### Repro Steps & Context

1. On Windows 11, open VS (repro with Version 17.8.0 Preview 2.0 [34020.76.main])
2. Open the VS Options -> NuGet Package Manager -> Package Sources page.
3. Without hovering over the list of package sources, observe the styling of the scrollbar makes the arrow buttons hard to see.

![image](https://github.com/NuGet/Home/assets/49205731/71be4d83-98cd-42cd-b22a-371b8bb392eb)

4. While hovering the mouse over the scrollbar, observe that the arrow buttons become visible.
![image](https://github.com/NuGet/Home/assets/49205731/3f13cc03-1b02-4921-b75e-69cc31eba100)

### Verbose Logs

```shell
Does not repro with Windows 10.
```

**Comment by donnie-msft** [comment]
@v-luzh could you watch for this bug as well, and let me know if there's other VS/system environment configurations you test with which does not repro the same issue? Thank you.

**Comment by v-luzh** [comment]
Hi @donnie-msft, it reproes for us on VS Main\34025.358 + NuGet Client Dev\6.8.0.83 with OS: Windows-11-Enterprise-22H2. 
But we thought it would be by-design, since the behavior of the scrollbar on the left of the screenshot below is hide too when not hovering on.
![image](https://github.com/NuGet/Home/assets/20632625/1f1a123d-870e-4195-9ce6-d37a67ae18f4)

**Comment by donnie-msft** [comment]
Thank you! I found it's actually reproing on all of Win11 and have reached out to the Windows team.
